proto/content_analysis/sdk/analysis.proto

```diff
@@ -8,24 +8,24 @@ option optimize_for = LITE_RUNTIME;
 
 package content_analysis.sdk;
 
-// Handshake enum is sent with every message indicating the
-// status of the issuing browser or the agent.
+// A Handshake is sent when opening a new connection from browser to agent.
 message Handshake {
-  enum Status {
-    FAILURE = 0;
-    SUCCESS = 1;
-  }
-  optional Status status = 1; 
+  // Process ID of the browser initiating the session.
+  optional int32 process_id = 1;
+  // Indicates whether a content analysis request will be sent in this session,
+  // following the handshake. If set to True, then the browser is expecting to
+  // receive a verdict response from the agent in this session, following
+  // the request. If set to False, the browser is performing a startup check or
+  // an intermittent heartbeat, and no response is expected from the agent.
+  optional bool content_analysis_request = 2;
 }
 
-// An Acknowledgment is sent from the browser to the agent
-// indicating whether the request was succeeded or failed.
+// An Acknowledgment is sent by the browser following the receipt of a response
+// from the agent.
 message Acknowledgement {
-  enum Status {
-    FAILURE = 0;
-    SUCCESS = 1;
-  }
-  optional Status status = 1;
+  // Indicates whether a verdict was successfully acted on by the browser.
+  // This should be logged by agents, especially if it's False.
+  optional bool verdict_success = 1;
```

Comment:
Same here, verdict_success -> verdict_succeeded  or  verdict_acted_on

Reply:
Changed to verdict_received.